src/adc.rs

```diff
@@ -20,7 +20,7 @@ impl ADCLine {
 
     /// Initialize an ADC line identified by the line number it is assigned on the board
     ///
-    /// Safety: See [Self::init()]
+    /// Safety: See [.init()](Self::init())
```

Comment:
In my experience, using `[text][target]` is more stable, because sometimes the target gets interpreted as a hyperlink, which does not happen with `[…][…]`. <https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html> mentions the `[…][…]` syntax, so it's no accident that it works.

Reply:
Thanks, forcepushed.